content/applications/inventory_and_mrp/purchase/manage_deals/calls_for_tenders.rst

```diff
@@ -0,0 +1,139 @@
+=================
+Calls for Tenders
+=================
+
+**Calls for Tenders** are one of the two main types of **Purchase Agreements** available by default
+in Odoo.
+
+*Calls for Tenders* are used to request offers from multiple vendors at the same time, by inviting
+those vendors to submit offers for similar goods or services all at once. This helps companies that
+use this feature to select the cheapest and fastest vendors in the market, depending on their
+needs.
+
+While calls for tenders are primaily used by organizations in the public sector who are legally
+bound to use them when making a purchase, private companies can also use them to spend efficiently,
+as well.
+
+When multiple offers are sent to a company, they can pick the one that works best for them,
+minimizing costs and increasing profitability in the most efficient manner.
+
+.. note::
+    To create and use calls for tenders, *Purchase Agreements* must be activated by going to
+    :menuselection:`Purchase --> Configuration --> Settings`.
+
+Example flow: Create a call for tender
+======================================
+
+To create a call for tender, first make sure that *Purchase Agreements* are enabled. Then, navigate
+to the :guilabel:`Purchase` application.
+
+.. image:: calls_for_tenders/calls-for-tenders-settings-page.png
+   :align: center
+   :alt: Purchase Agreements enabled in the Purchase app settings.
+
+.. tip::
+    To save time on a call for tender, custom vendors, prices, and delivery lead times can be set
+    in the :guilabel:`Purchase` tab on a product form.
+
+Create a :abbr:`RFQ (request for quotation)`
+--------------------------------------------
```

Comment:
Create an :abbr:`RFQ (request for quotation)`
-------------------------------------------------